### PR TITLE
Add back elixir tests to make check

### PR DESCRIPTION
**Pull request by garrensmith**
## Overview

Add back the elixir tests to make check. I want to use this branch to try and fix any flaky tests.

## Testing recommendations

Elixir tests should pass more consistently 

## Related Issues or Pull Requests

https://github.com/apache/couchdb/issues/1902

## Checklist

- [ ] Code is written and works correctly;
- [ ] Changes are covered by tests;
- [ ] Documentation reflects the changes;

### CODE REVIEW

**Comment by jiangphcn** [comment]
I will try after travis is happy with this PR.

File: test/elixir/test/replication_test.exs [hunk 1]
```diff
@@ -9,9 +9,9 @@ defmodule ReplicationTest do
   # TODO: Parameterize these
   @admin_account "adm:pass"
   @db_pairs_prefixes [
-    {"local-to-local", "", ""},
-    {"remote-to-local", "http://127.0.0.1:15984/", ""},
-    {"local-to-remote", "", "http://127.0.0.1:15984/"},
+    # {"local-to-local", "", ""},
+    # {"remote-to-local", "http://127.0.0.1:15984/", ""},
+    # {"local-to-remote", "", "http://127.0.0.1:15984/"},
```
Comment (jiangphcn) [inline test/elixir/test/replication_test.exs]: Is there reason why above 3 pairs are disabled for now?  Do we have flaky failures for these?

Comment (garrensmith) [reply test/elixir/test/replication_test.exs]: oops that was for testing. I'll add them back

**Comment by wohali** [comment]
@garrensmith you broke Jenkins.

Good place to start: https://builds.apache.org/blue/organizations/jenkins/CouchDB/detail/master/499/pipeline

Some of those are being killed because test runs are taking > 1h. Ycan relax that delay in Jenkinsfile but that's not great that our tests are taking 1h

**Pull request merged**